inc/search.class.php

```diff
@@ -1272,238 +1277,235 @@ static function constructData(array &$data, $onlycount = false) {
          );
       }
 
-      if ($result) {
```

Comment:
From this line to the end of the method : indentation level decreased. (diff unreadable)